WalletWasabi/JsonConverters/UnblindedSignatureJsonConverter.cs

```diff
@@ -6,16 +6,10 @@
 
 namespace WalletWasabi.JsonConverters;
 
-public class UnblindedSignatureJsonConverter : JsonConverter
+public class UnblindedSignatureJsonConverter : JsonConverter<UnblindedSignature>
 {
 	/// <inheritdoc />
-	public override bool CanConvert(Type objectType)
-	{
-		return objectType == typeof(UnblindedSignature);
-	}
-
-	/// <inheritdoc />
-	public override object ReadJson(JsonReader reader, Type? objectType, object? existingValue, JsonSerializer? serializer)
+	public override UnblindedSignature? ReadJson(JsonReader reader, Type objectType, UnblindedSignature? existingValue, bool hasExistingValue, JsonSerializer serializer)
```

Comment:
I would change L26 from:

```cs
private BigInteger StringToBigInteger(string num)
```

to

```cs
private BigInteger StringToBigInteger(string? num)
```

to fix two warnings.

(Alternatively we can put null checks in `ReadJson`.)

Reply:
Done.